exp/addons/internal/controllers/clusterresourceset_helpers.go

```diff
@@ -228,3 +230,33 @@ func computeHash(dataArr [][]byte) string {
 	}
 	return fmt.Sprintf("sha256:%x", hash.Sum(nil))
 }
+
+func getDataListAndHash(resourceKind string, unstructuredData map[string]interface{}) ([][]byte, string, []error) {
+	// Since maps are not ordered, we need to order them to get the same hash at each reconcile.
+	keys := make([]string, 0)
+
+	for key := range unstructuredData {
+		keys = append(keys, key)
+	}
+	sort.Strings(keys)
+
+	dataList := make([][]byte, 0)
+	errList := []error{}
```

Comment:
did you mean to initialize here also?

Reply:
I might be misunderstanding your question...

`errList` is initialized here, using the "literal" syntax, as opposed to the `make` built-in. But even if I declared `var errList []error`, it would be initialized, and I could use it in the `append` call on L248.

Reply:
It was a question of consistency. Not going to block over this.

Reply:
Got it.

The whole code base is inconsistent, to be honest. Sometimes `make`, other times the literal syntax. They're equivalent. I prefer using the simpler, literal syntax, unless the length is greater than zero.

Reply:
I believe they are not exactly equivalent, it has been admittedly a while, but I believe the difference is a question about initializing an array vs not initializing one but being declared.